duckduckgo/Android
# Broken site report snackbar

Task/Issue URL: https://app.asana.com/0/0/1208820300773406/f

### Description

Fix: Shows snackbar when submitting broken site report

### Steps to test this PR

1. Go to App Menu > Report Broken Site
2. Follow through report flow steps and submit
3. Report flow screen should disappear and a toast should appear saying "Thank you! Your report helps make DuckDuckGo better for everyone!"
